TeensyDmx.h

```diff
@@ -135,11 +143,54 @@ class TeensyDmx
         setChannels(startAddress - 1, values, length);
     }
 
+    void sendRDMDiscMute(byte *uid);
```

Comment:
Having to pass the UID into all of these functions seems a little tedious.  Is there not a better way to do it?

Reply:
Not really, as a controller there can be up to 32 responders on the line (or potentially more if RDM splitters are present).

We could want to control any, or all of them, hence needing to pass the UID each time, depending on which fixture we're interested in controlling. It actually gets a bit worse than this, as there's also the concept of sub-devices (e.g. dimmers in a rack), so I was thinking I really ought to add the ability to control that too.

I guess we could add some helper functions where you set the UID globally, and then it uses that for all subsequent calls.

Reply:
Perhaps a sub-class that gets constructed using a back-reference to the parent and a UID?

Reply:
Sounds like a good solution, I think one for V2 though.